scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -860,10 +913,19 @@ void BPF_STRUCT_OPS(rusty_stopping, struct task_struct *p, bool runnable)
 {
 	struct task_ctx *taskc;
 
-	if (fifo_sched)
+	if (!(taskc = lookup_task_ctx(p)))
 		return;
```

Comment:
Can we `scx_ops_error()` here?

Reply:
It also seems like other places where we do this check in rusty, we don't log the erorr.